### ISSUE TITLE
Errors from hooks creation are hard to read

### ISSUE THREAD

**Issue by imbstack** [taskcluster/taskcluster#1936]
**Describe the bug**
If a scopes error causes a hook to not fire, the error display is hard to read.

**To Reproduce**

1. Go to https://firefox-ci-tc.services.mozilla.com/hooks/project-relman/code-coverage-cron-production
2. Click on the "i" under "Fire Status" for `UTdd8xNrSC-7Ouams4fRHQ`
3. See that it is hard to read the error

**Expected behavior**
This is a structured error with one section of it being markdown for display. 

**Screenshots**

![image](https://user-images.githubusercontent.com/127521/68534987-2b4b8e80-02f0-11ea-976b-c140eca12304.png)

**Comment by djmitche** [comment]
This won't be a UI fix -- the service should be better about what it puts in that field.

**Comment by naima-shaikh** [comment]
@helfi92  May i take this one ?

**Comment by helfi92** [comment]
@imbstack @djmitche Do you know if this was fixed? I remember seeing a possible fix a while ago but I'm not certain.

**Comment by djmitche** [comment]
I don't believe so.  The fix would be somewhere around [here](https://github.com/taskcluster/taskcluster/blob/0e63621bee2dfaf166f7e56ad04dc1ccb623b847/services/hooks/src/taskcreator.js#L138-L154).
